Assets/Code/Scripts/SticksTmp.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+
+public class SticksTmp : MonoBehaviour
+{
+    private PlayerStatus status;
+    public TextMeshPro amount;
+
+    void Start()
+    {
+        status = FindObjectOfType<PlayerStatus>();
+    }
+
+    void Update()
+    {
+        int SticksAmount = status.Sticks;
+        amount.text = SticksAmount.ToString();
```

Comment:
Podobna sytuacja jak przy pliku 'TimerTmp.cs', nie warto rozbijać na nową klasę jednej praktycznie linijki do aktualizacji HUDu, wrzuć to do PlayerStatusu

Kolejnym problemem jest to, że funkcja Update będzie odświeżała wynik co klatkę, gdzie ta zmiana następuje raz na kolizję. Warto to wrzucić do funkcji, która inkrementuje liczbę patyków